### DISCUSSION TITLE
Math Equations in multiple-line \[  G= 2 + G \]  and inline \( G \)

### DISCUSSION THREAD

**Discussion by sunweihunu** [marp-team/marp#541] [Q&A] [answered]
How does Marp support Math Equations like the format as follows:
,multiple line 

\[  
G= 2 + G 
\]  

and inline 
\( G \)

**Comment by yhatt** [comment, marked as answer]
https://github.com/marp-team/marp-core#math-typesetting

**Reply by ameresv** [reply]
Just my two cents: Has the team considered implementing Typst in Marp? Is there any suggested workaround? I work with a lot of math equations and Marp, so it would be awesome to work with both in an integrated workflow. 
Thanks in advance.
Regards

**Reply by yhatt** [reply]
@ameresv Try [markdown-it-typst](https://github.com/xecades/markdown-it-typst) ([`@xecades/markdown-it-typst`](https://www.npmjs.com/package/@xecades/markdown-it-typst)) with Marp CLI. Currently it seems to work well.

```javascript
// engine.mjs
import markdownItTypst from '@xecades/markdown-it-typst'

export default ({ marp }) => marp.use(markdownItTypst)
```

```bash
npm install --save @marp-team/marp-cli @xecades/markdown-it-typst
npx marp --engine engine.mjs marp-typst.md
```

````markdown
# Typst in Marp

> ```typst
> #set text(font: "New Computer Modern")
> 
> Let $a$, $b$, and $c$ be the side
> lengths of right-angled triangle.
> Then, we know that:
> $ a^2 + b^2 = c^2 $
> 
> Prove by induction:
> $ sum_(k=1)^n k = (n(n+1)) / 2 $
> ```
>
> *— https://github.com/xecades/markdown-it-typst?tab=readme-ov-file#usage*
````

<details>
<summary>Screenshot</summary>

![marp-typst](https://github.com/user-attachments/assets/d3776d5b-fdc6-4133-a0fb-ce6774b19a19)

</details>

**Reply by ameresv** [reply]
Thanks for the support. I did not note that extension. I would like to know if there is a chance to implement it in the VS Code extension or just works using the terminal? I am pretty new at this so neither google or chatgpt gave me a good option for doing that. 
Thanks again
Regards

**Reply by yhatt** [reply]
Using markdown-it (or Marp) plugins is only available in Marp CLI because that is an advanced usage. VS Code extension cannot use plugins.
